contract/src/main/kotlin/com/ritense/valtimo/contract/authorization/UserManagementServiceHolder.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.ritense.authorization
+package com.ritense.valtimo.contract.authorization
```

Comment:
This class and the other classes don't belong in the contract module

Reply:
I discussed this with @ivo-ritense, there was no other reasonable option it seems

Reply:
I think it's hard to get around some of this. it's a choice between having classes like this in contract, or having core be dependent on dashboard, which also isn't great. 

The classes that are in contract now are either general configuration classes, or classes that are meant to be re-usable between data sources.